tfrecorder/input_schema.py

```diff
@@ -0,0 +1,163 @@
+# Lint as: python3
+
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Defines input types for TFRecorder's input schema."""
+
+from collections import OrderedDict
+from dataclasses import dataclass
```

Comment:
nit: import up to package/model level: https://google.github.io/styleguide/pyguide.html#22-imports

Find and replace can ease the pain of `@dataclasses.dataclass`.

Reply:
done